packages/mandelbrot/assets/scss/components/_file-browser.scss

```diff
@@ -31,36 +18,37 @@
     font-weight: bold;
 }
 
-.select2-selection.select2-selection--single {
-    border-color: $color-frame-border;
-
-    .select2-container--focus & {
-        --border-color: rgb(var(--skin-links));
-
-        outline: none;
-        border-color: $color-link;
-        border-color: var(--border-color, $color-link);
-    }
-}
-
-.select2-results__option {
-    @include font(caption);
-
+.FileBrowser-select {
+    position: relative;
+    appearance: none;
     min-width: 15em;
-
-    &[aria-selected="true"] {
-        background-color: rgba($color-frame-border, 0.1) !important;
+    padding: 0 24px 0 8px;
+    border: 1px solid rgba(83, 83, 99, 0.25);
+    border-radius: 4px;
+    font-size: 0.875rem;
+    color: #444;
+    line-height: 28px;
+    height: 28px;
+    cursor: pointer;
+    background-color: #fff;
+    background-image: url("data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%22292.4%22%20height%3D%22292.4%22%3E%3Cpath%20fill%3D%22%23888888%22%20d%3D%22M287%2069.4a17.6%2017.6%200%200%200-13-5.4H18.4c-5%200-9.3%201.8-12.9%205.4A17.6%2017.6%200%200%200%200%2082.2c0%205%201.8%209.3%205.4%2012.9l128%20127.9c3.6%203.6%207.8%205.4%2012.8%205.4s9.2-1.8%2012.8-5.4L287%2095c3.5-3.5%205.4-7.8%205.4-12.8%200-5-1.9-9.2-5.5-12.8z%22%2F%3E%3C%2Fsvg%3E");
+    background-repeat: no-repeat;
+    background-position: right 0.5em top 55%;
+    background-size: 0.65em auto;
+
+    &::-ms-expand {
+        display: none;
     }
-}
 
-.select2-results__option--highlighted[aria-selected] {
-    --background-color: rgb(var(--skin-accent));
-    --color: rgb(var(--skin-complement));
+    &::-ms-value {
+        color: inherit;
+        background: inherit;
+    }
 
-    background-color: $color-header-background !important;
-    background-color: var(--background-color, $color-header-background) !important;
-    color: $color-header-content !important;
-    color: var(--color, $color-header-content) !important;
+    &:focus {
+        outline: none;
```

Comment:
I wouldn’t suppress the browser default without adding a large box-shadow instead, border isn’t enough to me to be easily found in the page. To be sync with the search field as well I think.

Reply:
I removed the custom focus styles completely. I played around with custom because I happened to test in IE11 and there were no focus styles visible at all... then again, the search input also does not have focus styles in IE11. Maybe it's not a too bad to leave it a bit buggy in IE?

Reply:
Yeah no need to workaround that for a browser that will be considered dead in 6 months.